### ISSUE TITLE
macOS x86_64 do not work anymore

### ISSUE THREAD

**Issue by sarlinpe** [colmap/colmap#2728]
Running `import pycolmap` on a macOS 11.4 x86_64 machine with wheels built by the CI yields:
```
ERROR: something wrong with flag 'timestamp_in_logfile_name' in file '/Users/runner/work/vcpkg/buildtrees/glog/src/v0.7.1-795557b621.clean/src/flags.cc'. 
One possibility: file '/Users/runner/work/vcpkg/buildtrees/glog/src/v0.7.1-795557b621.clean/src/flags.cc' is being linked both statically and dynamically into this executable.
```
It seems to have been introduced in https://github.com/colmap/colmap/commit/03556bb2cc941b58c3a4c244e5549e06c2f8117a but right now I'm lacking time to investigate.

**Comment by B1ueber2y** [comment]
Hey. Do you have pyceres installed? And have you updated to the new version? Not sure if this is relevant: https://github.com/cvg/pyceres/pull/52

**Comment by sarlinpe** [comment]
Indeed, the error precisely seems to occur because pyceres is installed (v2.3 in my case). Running `import pyceres` works fine. Uninstalling pyceres makes pycolmap work as expected.

**Comment by ahojnnes** [comment]
Closing: pyceres version incompatibility; tracked in pyceres repo.
